css/mark-nowowiejski.css

```diff
@@ -0,0 +1,25 @@
+@import url(https://fonts.googleapis.com/css?family=Ubuntu);
+
+body {
+  font-family: Ubuntu;
+}
+
+h1 {
+  color: red;
+}
+#first-bullet{
+  color:green;
+}
+
+.bi {
+  font-weight: bold;
+  font-style: italic;
+}
+
+ul > li {
```

Comment:
Check out the updated task for using a `child selector` in this assignment, and update this CSS rule.

Awesome work on this Mark!